src/source/generator.scala

```diff
@@ -137,6 +138,22 @@ package object generatorTools {
     }
   }
 
+  object JavaAccessModifier extends Enumeration {
+    val Public = Value("public")
+    val Protected = Value("protected")
+    val Package = Value("package")
+
+    def getCodeGenerationString(javaAccessModifier: JavaAccessModifier.Value): String = {
+      javaAccessModifier match {
+        case Public => "public "
+        case Protected => "protected "
+        case Package => ""
```

Comment:
In other cases where package-private is used by Djinni (e.g. in constant fields) the generator outputs a comment /*package*/ in place of the accessibility.